server/middleware/validateParty.js

```diff
@@ -1,31 +1,31 @@
 import HelperUtils from '../utility/helperUltis';
-import partiesDb from '../models/partyModels';
+import partyDb from '../model/partyModel';
 
 
 /**
- * @class ValidateParties
+ * @class ValidateParty
  * @description Intercepts and validates a given request for parties endpoints
- * @exports ValidateParties
+ * @exports ValidateParty
  */
 
-class ValidateParties {
+export default class ValidateParty {
   /**
          * @description Get a specific party by id
          * @param {object} req - The request object
          * @param {object} res - The response object
          * @param {function} next - Calls the next function
          * @returns {object} JSON representing the failure message
-         * @memberof ValidateParties
+         * @memberof ValidateParty
          */
-  static findPartiesById(req, res, next) {
+  static findPartyById(req, res, next) {
     const { id } = req.params;
     if (!Number(id)) {
       return res.status(400).json({
```

Comment:
Status code should be 404 for non-existing values.

Reply:
Acknowledged and resolved